types/index.ts

```diff
@@ -3,7 +3,7 @@ export interface VerdaccioOptions {
   url_prefix: string;
   base: string;
   language?: string;
-  darkMode?: boolean;
+  dark_mode?: boolean;
```

Comment:
I prefer cammelCase, hopefully we can remove some day all the `_` ...

Reply:
me too, but then I noticed that we have `url_prefix` and I decided to do like this

Reply:
`url_prefix` is from the *sinopia* era, the previous author was quite all over the place using that, it took me few years to get rid of those, but some configurations are still using `_` due backward compatibility unfortunately.